src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -257,16 +245,47 @@ export function GET(context) {
 }
 ```
 
-To access the value of the `MY_VARIABLE` binding add the following to your code:
+See the [list of all supported bindings](https://developers.cloudflare.com/workers/wrangler/api/#supported-bindings) in the Cloudflare documentation.
+
+
+### Environment variables and secrets
+
+The Cloudflare runtime treats environment variables as a type of binding.
+
+For example, you can define an [environment variable](https://developers.cloudflare.com/workers/configuration/environment-variables/#add-environment-variables-via-wrangler) in `wrangler.json` as follows:
+
+```json title="wrangler.json"
+{
+  "vars" : {
+    "MY_VARIABLE": "test"
+  }
+}
+```
+
+Secrets are a special type of environment variable that allow you to attach encrypted text values to your Worker. They need to be defined differently to ensure they are not visible after you set them.
+To define `secrets`, add them through the [Wrangler CLI](https://developers.cloudflare.com/workers/wrangler/) rather than in your Wrangler config file. 
```

Comment:
```suggestion

To define `secrets`, add them through the [Wrangler CLI](https://developers.cloudflare.com/workers/wrangler/) rather than in your Wrangler config file. 
```
Let's make this a new paragraph for readability!